include/ht_scene.h

```diff
@@ -79,6 +82,16 @@ namespace Hatchit {
              */
             void Unload(void);
 
+            /**
```

Comment:
Place static functions at top of public class definition.  Put documentation in source file.